cmd/ctl/go.mod

```diff
@@ -12,11 +12,11 @@ go 1.23.8
 // or a branch name (master).
 
 require (
-	github.com/cert-manager/cert-manager v1.12.16
+	github.com/cert-manager/cert-manager v1.12.17-0.20250423105235-3b9187a6256a
```

Comment:
I first changed the version to `release-1.12` by hand, and then ran `make tidy` to turn that into a pseudo Go module version.

This will be changed again during the 1.12 release.